src/CMakeLists.txt

```diff
@@ -262,18 +262,11 @@ install(FILES ${valhalla_hdrs}
   COMPONENT development)
 
 if(PKG_CONFIG_FOUND)
-  ## Configure libvalhalla.pc file with valhalla target linking options via PkgConfig linker
-  set(CMAKE_PkgConfig_LINK_EXECUTABLE "<CMAKE_COMMAND> -DINPUT=${VALHALLA_SOURCE_DIR}/libvalhalla.pc.in -DOUTPUT=<TARGET> -DVERSION=${VERSION} -Dprefix=${CMAKE_INSTALL_PREFIX} -Dexec_prefix=${CMAKE_INSTALL_PREFIX} -Dlibdir=${CMAKE_INSTALL_LIBDIR} -Dincludedir=${CMAKE_INSTALL_INCLUDEDIR} -Ddeplibs=\"<FLAGS> <LINK_FLAGS> <LINK_LIBRARIES>\" -P ${CMAKE_SOURCE_DIR}/cmake/PkgConfig.cmake")
-
-  add_executable(libvalhalla.pc EXCLUDE_FROM_ALL ../libvalhalla.pc.in)
-  target_link_libraries(libvalhalla.pc valhalla)
-  set_target_properties(libvalhalla.pc PROPERTIES LINKER_LANGUAGE PkgConfig)
-  set_target_properties(libvalhalla.pc PROPERTIES FOLDER "Library")
-  install(CODE "execute_process(COMMAND ${CMAKE_COMMAND} --build . --target libvalhalla.pc OUTPUT_QUIET ERROR_VARIABLE _err RESULT_VARIABLE _res)
-  if(NOT \${_res} EQUAL 0)
-    message(FATAL_ERROR \"Configuring libvalhalla.pc failed: \${_err}\")
-  endif()")
-  install(FILES ${VALHALLA_BINARY_DIR}/libvalhalla.pc
-    DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig"
-    COMPONENT development)
+  set(DEPLIBS "<FLAGS> <LINK_FLAGS> <LINK_LIBRARIES>")
```

Comment:
This is going wrong, but I don't fully understand why it worked in the previous method, investigating...

Reply:
For now I've set it hard-coded to the expected output, not sure if this can be improved somehow...